### ISSUE TITLE
Suggestion for remote management on Android 

### ISSUE THREAD

**Issue by linlinxza** [LubosD/fatrat#30]
Could an app be designed so that links can be shared to the download manager? I find that this idea would make it more intuitive on mobile devices instead of using a Web browser.

**Comment by LubosD** [comment]
I fully agree, unfortunately I haven't yet had enough time to work on one.
